resources/lang/en.json

```diff
@@ -2255,6 +2255,9 @@
   "Should NOT have more than {max} items" : "Should NOT have more than {max} items",
   "Must have at least {min}" : "Must have at least {min}",
   "Invalid type" : "Invalid type",
+  "The Two Step Authentication Method has not been set. Please contact your administrator.": "The Two Step Authentication Method has not been set. Please contact your administrator.",
```

Comment:
Consider to add the label to the other language files like `es.json`

Reply:
the other languages ​​are generated differently so it is not necessary to add it now.